libraries/api/src/middleware/auth.js

```diff
@@ -130,22 +122,22 @@ const remoteAuthentication = ({signatureValidator, entry}) => {
     // so this is probably a fair policy for now. We can always allow more.
     if (req.headers && req.headers.authorization &&
         req.query && req.query.bewit) {
-      return Promise.resolve({
+      return {
```

Comment:
We used to return a `Promise`, and now just an object - is that ok?

Reply:
I'll go through and double check to make sure. I definitely don't want to break this part.

Reply:
Ah, there was one `signatureValidator` that wasn't an `async`! I've updated it to be one and also brought back in the `Promise.resolve` for that specific part just in case.

Nice catch.